packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -1,4 +1,78 @@
-{{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  {{!-- {{yield}} --}}
+
+  {{! Hds-TabList }}
+  <div class="hds-tab-list">
+    <ul class="hds-tab-list__list" role="tablist">
+      {{! Hds-Tab }}
+      <li class="hds-tab" role="presentation">
+        <a
```

Comment:
```suggestion
        <a
          class="hds-tabs__tab"
```

[suggestion] Since this doesn't have a presentational role, but a functional role, it should have a class name. @didoo might have stronger opinions on what it is, `hds-tabs__tab` or `hds-tabs__tablist__tab`, not 100% sure but there will be patterns in the other components to follow.

Reply:
Thanks, I'm going to break these out into separate child components next so that may affect the naming.